cmd/kubeadm/app/util/audit/utils.go

```diff
@@ -65,3 +65,31 @@ func writePolicyToDisk(policyFile string, policy *auditv1beta1.Policy) error {
 
 	return nil
 }
+
+// CreateDefaultAuditWebhookConfig writes the default audit webhook config to disk.
+func CreateDefaultAuditWebhookConfig(webhookConfigFile string) error {
+	// We do not have a default, but we do have a volume mapping for this file
+	return os.OpenFile(name, os.O_RDONLY|os.O_CREATE, 0644)
+}
+
+func writeWebhookConfigToDisk(webhookConfigFile string, policy *auditv1beta1.Policy) error {
```

Comment:
I might have missed something but I think you can reuse the `writePolicyToDisk` function as the only thing I can see that has changed is the name of the variable `policyFile` variable?

Reply:
I need to refactor a bit, the Policy is of a type policy, I think an AuditWebhookConfig is of [type Config](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go#L53)

From https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#use-logstash-to-collect-and-distribute-audit-events-from-webhook-backend:

```yaml
apiVersion: v1
kind: Config
clusters:
- cluster:
    server: http://<ip_of_logstash>:8888
  name: logstash
contexts:
- context:
    cluster: logstash
    user: ""
  name: default-context
current-context: default-context
preferences: {}
users: []
```